src/crawlee/_utils/docs.py

```diff
@@ -0,0 +1,2 @@
+def docs_group(group_name: str) -> None:
+    pass
```

Comment:
This likely breaks all the code and needs fixing. 

All the "noop" decorator approaches (that work for both classes and functions) I found had way too many lines - definitely more than I'm comfortable with changing :) 

Can one of you Python whizzes fix this please? (cc @vdusek @Pijukatel )

Reply:
Something like this should work:

```py
class docs_group:
  def __init__(self, group_name: str) -> None:
    pass

  def __call__(self, callable: Callable) -> Callable:
    return callable
```

You could also make a function that returns a function, but I tend to find that a class is more readable.